paasta_tools/mesos_tools.py

```diff
@@ -137,6 +137,17 @@ def filter_not_running_tasks(tasks):
     return [task for task in tasks if task['state'] != 'TASK_RUNNING']
 
 
+def get_running_task_ids_from_mesos_slave():
```

Comment:
Nice! I was surprised there wasn't already a way in place to do this. Would it be an idea to make job_id default to `''` on any function where it can be used in this way in that case?